python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst

```diff
@@ -7,73 +7,37 @@ The importance factors derived from a Taylor expansion are
 defined to rank the sensitivity of the inputs to
 the output for central dispersion analysis.
 
-Let us denote by :math:`\uX` the input random vector.
-Assume that the marginals of :math:`\uX` are independent.
-Suppose that :math:`Z = h(\ux)` is a real function of the
-input, i.e. :math:`n_Z = 1`.
-Assume that the order 1 Taylor expansion of the function :math:`h`
-at the point :math:`\ux = \muX` is exact, i.e.:
+We denote by :math:`\uX` the input random vector. We assume that the marginals of :math:`\uX` are independent.
```

Comment:
```suggestion
Let :math:`\uX` be the input random vector. We assume that the marginals of :math:`\uX` are independent.
```
At some point you will get used to the "Let X be a random variable." english expression :)

Reply:
There is no need to insert a new line after each sentence:
![image](https://github.com/openturns/openturns/assets/31351465/cf6e25d6-6c29-4a99-a0f8-d035131e7160)

Reply:
yes...

Reply:
ok